build.sbt

```diff
@@ -306,7 +306,7 @@ lazy val testsInput = project
     scalacOptions += s"-P:semanticdb:sourceroot:${sourceDirectory.in(Compile).value}",
     scalacOptions ~= (_.filterNot(_ == "-Yno-adapted-args")),
     scalacOptions += "-Ywarn-adapted-args", // For NoAutoTupling
-    scalacOptions += "-Ywarn-unused-import", // For RemoveUnusedImports
+    scalacOptions += "-Ywarn-unused", // For RemoveUnusedImports and RemoveUnusedTerms
```

Comment:
Seems we need to keep `-Ywarn-unused-import` for 2.11 https://travis-ci.org/scalacenter/scalafix/jobs/278586505